### ISSUE TITLE
Training does not compute after first iteration

### ISSUE THREAD

**Issue by 7heRedFox** [DeepLabCut/DeepLabCut#1762]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

I try to train a network but the program freeze
![image](https://user-images.githubusercontent.com/48990624/162236090-3a0c28b8-045a-4e4e-866a-cf0a57d6f24b.png)
As you can see the gpu is detected and it does few iterations but stops there almost instantly.
I tried every TF, cuda and cudnn trio possible and can't go further than that

### Operating System

operating system : win 10

### DeepLabCut version

dlc version : 
last (2.2.0.6)

conda env:
packages in environment at C:\Users\TeamNCMC\.conda\envs\dlc:

Name                    Version                   Build  Channel
absl-py                   1.0.0                    pypi_0    pypi
argon2-cffi               21.3.0             pyhd8ed1ab_0    conda-forge
argon2-cffi-bindings      21.2.0           py38h294d835_1    conda-forge
asttokens                 2.0.5              pyhd8ed1ab_0    conda-forge
astunparse                1.6.3                    pypi_0    pypi
attrs                     21.4.0             pyhd8ed1ab_0    conda-forge
backcall                  0.2.0              pyh9f0ad1d_0    conda-forge
backports                 1.0                        py_2    conda-forge
backports.functools_lru_cache 1.6.4              pyhd8ed1ab_0    conda-forge
beautifulsoup4            4.10.0             pyha770c72_0    conda-forge
bleach                    4.1.0              pyhd8ed1ab_0    conda-forge
bzip2                     1.0.8                h8ffe710_4    conda-forge
ca-certificates           2022.3.29            haa95532_0
cachetools                5.0.0                    pypi_0    pypi
certifi                   2021.10.8                pypi_0    pypi
cffi                      1.15.0           py38hd8c33c5_0    conda-forge
charset-normalizer        2.0.12                   pypi_0    pypi
clang                     5.0                      pypi_0    pypi
colorama                  0.4.4              pyh9f0ad1d_0    conda-forge
cudatoolkit               11.3.1               h59b6b97_2
cudnn                     8.2.1                cuda11.3_0
cycler                    0.11.0                   pypi_0    pypi
debugpy                   1.5.1            py38h885f38d_0    conda-forge
decorator                 4.4.2                    pypi_0    pypi
deeplabcut                2.2.0.6                  pypi_0    pypi
defusedxml                0.7.1              pyhd8ed1ab_0    conda-forge
entrypoints               0.4                pyhd8ed1ab_0    conda-forge
executing                 0.8.3              pyhd8ed1ab_0    conda-forge
ffmpeg                    4.3.1                ha925a31_0    conda-forge
filterpy                  1.4.5                    pypi_0    pypi
flatbuffers               2.0                      pypi_0    pypi
flit-core                 3.7.1              pyhd8ed1ab_0    conda-forge
fonttools                 4.31.2                   pypi_0    pypi
gast                      0.5.3                    pypi_0    pypi
google-auth               2.6.2                    pypi_0    pypi
google-auth-oauthlib      0.4.6                    pypi_0    pypi
google-pasta              0.2.0                    pypi_0    pypi
grpcio                    1.44.0                   pypi_0    pypi
h5py                      3.6.0                    pypi_0    pypi
icu                       69.1                 h0e60522_0    conda-forge
idna                      3.3                      pypi_0    pypi
imageio                   2.16.1                   pypi_0    pypi
imageio-ffmpeg            0.4.5                    pypi_0    pypi
imgaug                    0.4.0                    pypi_0    pypi
importlib-metadata        4.11.3           py38haa244fe_1    conda-forge
importlib_resources       5.6.0              pyhd8ed1ab_0    conda-forge
ipykernel                 6.12.1           py38h4317176_0    conda-forge
ipython                   8.2.0            py38haa244fe_0    conda-forge
ipython_genutils          0.2.0                      py_1    conda-forge
ipywidgets                7.7.0              pyhd8ed1ab_0    conda-forge
jedi                      0.18.1           py38haa244fe_1    conda-forge
jinja2                    3.1.1              pyhd8ed1ab_0    conda-forge
joblib                    1.1.0                    pypi_0    pypi
jpeg                      9e                   h8ffe710_0    conda-forge
jsonschema                4.4.0              pyhd8ed1ab_0    conda-forge
jupyter                   1.0.0            py38haa244fe_7    conda-forge
jupyter_client            7.2.1              pyhd8ed1ab_0    conda-forge
jupyter_console           6.4.3              pyhd8ed1ab_0    conda-forge
jupyter_core              4.9.2            py38haa244fe_0    conda-forge
jupyterlab_pygments       0.1.2              pyh9f0ad1d_0    conda-forge
jupyterlab_widgets        1.1.0              pyhd8ed1ab_0    conda-forge
keras                     2.8.0                    pypi_0    pypi
keras-preprocessing       1.1.2                    pypi_0    pypi
kiwisolver                1.4.2                    pypi_0    pypi
libclang                  13.0.0                   pypi_0    pypi
libffi                    3.4.2                h8ffe710_5    conda-forge
libpng                    1.6.37               h1d00b33_2    conda-forge
libsodium                 1.0.18               h8d14728_1    conda-forge
libzlib                   1.2.11            h8ffe710_1014    conda-forge
llvmlite                  0.38.0                   pypi_0    pypi
markdown                  3.3.6                    pypi_0    pypi
markupsafe                2.1.1            py38h294d835_1    conda-forge
matplotlib                3.5.1                    pypi_0    pypi
matplotlib-inline         0.1.3              pyhd8ed1ab_0    conda-forge
mistune                   0.8.4           py38h294d835_1005    conda-forge
moviepy                   1.0.3                    pypi_0    pypi
msgpack                   1.0.3                    pypi_0    pypi
msgpack-numpy             0.4.7.1                  pypi_0    pypi
nb_conda                  2.2.1            py38haa244fe_4    conda-forge
nb_conda_kernels          2.3.1            py38haa244fe_1    conda-forge
nbclient                  0.5.13             pyhd8ed1ab_0    conda-forge
nbconvert                 6.4.5              pyhd8ed1ab_2    conda-forge
nbconvert-core            6.4.5              pyhd8ed1ab_2    conda-forge
nbconvert-pandoc          6.4.5              pyhd8ed1ab_2    conda-forge
nbformat                  5.3.0              pyhd8ed1ab_0    conda-forge
nest-asyncio              1.5.5              pyhd8ed1ab_0    conda-forge
networkx                  2.7.1                    pypi_0    pypi
notebook                  6.4.10             pyha770c72_0    conda-forge
numba                     0.55.1                   pypi_0    pypi
numexpr                   2.8.1                    pypi_0    pypi
numpy                     1.20.0                   pypi_0    pypi
oauthlib                  3.2.0                    pypi_0    pypi
opencv-python             4.5.5.64                 pypi_0    pypi
openssl                   1.1.1n               h2bbff1b_0
opt-einsum                3.3.0                    pypi_0    pypi
packaging                 21.3               pyhd8ed1ab_0    conda-forge
pandas                    1.4.2                    pypi_0    pypi
pandoc                    2.17.1.1             h57928b3_0    conda-forge
pandocfilters             1.5.0              pyhd8ed1ab_0    conda-forge
parso                     0.8.3              pyhd8ed1ab_0    conda-forge
patsy                     0.5.2                    pypi_0    pypi
pickleshare               0.7.5                   py_1003    conda-forge
pillow                    9.1.0                    pypi_0    pypi
pip                       22.0.4             pyhd8ed1ab_0    conda-forge
proglog                   0.1.9                    pypi_0    pypi
prometheus_client         0.14.0             pyhd8ed1ab_0    conda-forge
prompt-toolkit            3.0.29             pyha770c72_0    conda-forge
prompt_toolkit            3.0.29               hd8ed1ab_0    conda-forge
protobuf                  3.20.0                   pypi_0    pypi
psutil                    5.9.0            py38h294d835_1    conda-forge
pure_eval                 0.2.2              pyhd8ed1ab_0    conda-forge
pyasn1                    0.4.8                    pypi_0    pypi
pyasn1-modules            0.2.8                    pypi_0    pypi
pycparser                 2.21               pyhd8ed1ab_0    conda-forge
pygments                  2.11.2             pyhd8ed1ab_0    conda-forge
pyparsing                 3.0.7              pyhd8ed1ab_0    conda-forge
pyqt                      5.12.3           py38haa244fe_8    conda-forge
pyqt-impl                 5.12.3           py38h885f38d_8    conda-forge
pyqt5-sip                 4.19.18          py38h885f38d_8    conda-forge
pyqtchart                 5.12             py38h885f38d_8    conda-forge
pyqtwebengine             5.12.1           py38h885f38d_8    conda-forge
pyrsistent                0.18.1           py38h294d835_1    conda-forge
python                    3.8.13          h9a09f29_0_cpython    conda-forge
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python-fastjsonschema     2.15.3             pyhd8ed1ab_0    conda-forge
python_abi                3.8                      2_cp38    conda-forge
pytz                      2022.1                   pypi_0    pypi
pywavelets                1.3.0                    pypi_0    pypi
pywin32                   303              py38h294d835_0    conda-forge
pywinpty                  2.0.5            py38hd3f51b4_1    conda-forge
pyyaml                    6.0                      pypi_0    pypi
pyzmq                     22.3.0           py38h09162b1_2    conda-forge
qt                        5.12.9               h556501e_6    conda-forge
qtconsole                 5.3.0              pyhd8ed1ab_0    conda-forge
qtconsole-base            5.3.0              pyhd8ed1ab_0    conda-forge
qtpy                      2.0.1              pyhd8ed1ab_0    conda-forge
requests                  2.27.1                   pypi_0    pypi
requests-oauthlib         1.3.1                    pypi_0    pypi
rsa                       4.8                      pypi_0    pypi
ruamel-yaml               0.17.21                  pypi_0    pypi
ruamel-yaml-clib          0.2.6                    pypi_0    pypi
scikit-image              0.18.1                   pypi_0    pypi
scikit-learn              1.0.2                    pypi_0    pypi
send2trash                1.8.0              pyhd8ed1ab_0    conda-forge
setuptools                62.0.0                   pypi_0    pypi
shapely                   1.8.1.post1              pypi_0    pypi
six                       1.15.0                   pypi_0    pypi
soupsieve                 2.3.1              pyhd8ed1ab_0    conda-forge
sqlite                    3.37.1               h8ffe710_0    conda-forge
stack_data                0.2.0              pyhd8ed1ab_0    conda-forge
statsmodels               0.13.2                   pypi_0    pypi
tables                    3.6.1                    pypi_0    pypi
tabulate                  0.8.9                    pypi_0    pypi
tensorboard               2.8.0                    pypi_0    pypi
tensorboard-data-server   0.6.1                    pypi_0    pypi
tensorboard-plugin-wit    1.8.1                    pypi_0    pypi
tensorflow                2.8.0                    pypi_0    pypi
tensorflow-estimator      2.7.0                    pypi_0    pypi
tensorflow-io-gcs-filesystem 0.24.0                   pypi_0    pypi
tensorpack                0.11                     pypi_0    pypi
termcolor                 1.1.0                    pypi_0    pypi
terminado                 0.13.3           py38haa244fe_1    conda-forge
testpath                  0.6.0              pyhd8ed1ab_0    conda-forge
tf-estimator-nightly      2.8.0.dev2021122109          pypi_0    pypi
tf-slim                   1.1.0                    pypi_0    pypi
threadpoolctl             3.1.0                    pypi_0    pypi
tifffile                  2022.3.25                pypi_0    pypi
tk                        8.6.12               h8ffe710_0    conda-forge
tornado                   6.1              py38h294d835_3    conda-forge
tqdm                      4.64.0                   pypi_0    pypi
traitlets                 5.1.1              pyhd8ed1ab_0    conda-forge
typing-extensions         4.1.1                    pypi_0    pypi
ucrt                      10.0.20348.0         h57928b3_0    conda-forge
urllib3                   1.26.9                   pypi_0    pypi
vc                        14.2                 hb210afc_6    conda-forge
vs2015_runtime            14.29.30037          h902a5da_6    conda-forge
wcwidth                   0.2.5              pyh9f0ad1d_2    conda-forge
webencodings              0.5.1                      py_1    conda-forge
werkzeug                  2.1.1                    pypi_0    pypi
wheel                     0.37.1                   pypi_0    pypi
widgetsnbextension        3.6.0            py38haa244fe_0    conda-forge
wincertstore              0.2              py38haa95532_2
winpty                    0.4.3                         4    conda-forge
wrapt                     1.14.0                   pypi_0    pypi
wxpython                  4.0.7.post2              pypi_0    pypi
xz                        5.2.5                h62dcd97_1    conda-forge
zeromq                    4.3.4                h0e60522_1    conda-forge
zipp                      3.8.0              pyhd8ed1ab_0    conda-forge
zlib                      1.2.11            h8ffe710_1014    conda-forge

### DeepLabCut mode

single animal

### Device type

gpu : gtx 1080TI

### Steps To Reproduce

1. activate dlc
2. ipython
3. import deeplabcut as dlc
4. dlc.launch_dlc()
In gui
5. Load project, go to train tab
6. select shuffle 4 (the last one in date, made in the same environnement)
7. open config
8. close it after check
9. update parameters button
10. run
11. notice failure

### Relevant log output

```shell
PS C:\Users\TeamNCMC> nvidia-smi
Thu Apr  7 17:32:23 2022
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 512.15       Driver Version: 512.15       CUDA Version: 11.6     |
|-------------------------------+----------------------+----------------------+
| GPU  Name            TCC/WDDM | Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|                               |                      |               MIG M. |
|===============================+======================+======================|
|   0  NVIDIA GeForce ... WDDM  | 00000000:65:00.0  On |                  N/A |
| 23%   31C    P8    13W / 250W |  10471MiB / 11264MiB |      0%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+

+-----------------------------------------------------------------------------+
| Processes:                                                                  |
|  GPU   GI   CI        PID   Type   Process name                  GPU Memory |
|        ID   ID                                                   Usage      |
|=============================================================================|
|    0   N/A  N/A      1404    C+G                                   N/A      |
|    0   N/A  N/A      2716    C+G   ...perience\NVIDIA Share.exe    N/A      |
|    0   N/A  N/A      3880    C+G   ...y\ShellExperienceHost.exe    N/A      |
|    0   N/A  N/A      5696    C+G   ...5n1h2txyewy\SearchApp.exe    N/A      |
|    0   N/A  N/A      7788    C+G   C:\Windows\explorer.exe         N/A      |
|    0   N/A  N/A      7884    C+G   ...ekyb3d8bbwe\YourPhone.exe    N/A      |
|    0   N/A  N/A      8964    C+G   ...5n1h2txyewy\SearchApp.exe    N/A      |
|    0   N/A  N/A     12144    C+G   ...bbwe\Microsoft.Photos.exe    N/A      |
|    0   N/A  N/A     12540    C+G   ...lPanel\SystemSettings.exe    N/A      |
|    0   N/A  N/A     12796    C+G   ...2txyewy\TextInputHost.exe    N/A      |
|    0   N/A  N/A     13144    C+G   ...ram Files\Opera\opera.exe    N/A      |
|    0   N/A  N/A     13668    C+G   ...wekyb3d8bbwe\Video.UI.exe    N/A      |
|    0   N/A  N/A     14192    C+G   ...kyb3d8bbwe\Calculator.exe    N/A      |
|    0   N/A  N/A     15932      C                                   N/A      |
+-----------------------------------------------------------------------------+

PS C:\Users\TeamNCMC> nvcc -V
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2022 NVIDIA Corporation
Built on Tue_Mar__8_18:36:24_Pacific_Standard_Time_2022
Cuda compilation tools, release 11.6, V11.6.124
Build cuda_11.6.r11.6/compiler.31057947_0
```

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by MMathisLab** [comment]
hi @7heRedFox many issues about this, did you search? :) 

 Please check: (1) how large are your images (pixel x pixel size); (2) what batch size are you using? (3) you are using a very large network (resnet101, which we do not generally recommend: https://deeplabcut.github.io/DeepLabCut/docs/recipes/nn.html#what-neural-network-should-i-use-trade-offs-speed-performance-and-considerations)

**Comment by jeylau** [comment]
Possibly fixed by #1738

**Comment by 7heRedFox** [comment]
I did have a look but similar topic seemed old.
Moreover it worked just fine before with similar data but someone updated CUDA for some reason and I had to go through update hell to reach this point
1) image are 2384x2048 but res cannot really go down as the animal is far from the objective of the camera(and DLC handled it just fine before)
2) batch size is 8 in the config file but display 1 when launching training in the consol
3) I tested resnet50 as well as efficientnetb0 but with no success and resnet50 was not good enough last time I used it, I would like to use a efficientnetb7 as performance seems great but it's not the topic here

**Comment by 7heRedFox** [comment]
Ok I don't know how I missed that one but issue #1725 show me the solution
I upped the max size limit to 2300 to match my images, thx @jeylau for the hint
I was stuck in a "that looks like a TF dependency problem" for days

**Comment by 7heRedFox** [comment]
Quick thought : maybe a 'trying with more permissive parameter' warning and changing maw_size_limit to max(width,height) with an OOM handling could do the trick
